api/cookies/cookie_clearer/popup.js

```diff
@@ -0,0 +1,101 @@
+const form = document.getElementById("control-row");
+const go = document.getElementById("go");
+const input = document.getElementById("input");
+const message = document.getElementById("message");
+
+
+document.addEventListener("DOMContentLoaded", initPopupWindow);
```

Comment:
Why do this? Your script runs synchronously where it's loaded; all the DOM you care about is above this.

I'd rather you use `type="module"` (which is implicit defer) and then use top-level `await` for the setup. (Chrome's samples regularly have `async` handlers for events, which .... I get, we have async APIs, but is hard).

Reply:
This is an artifact of the original implementation. The page originally inserted popup.js in the head & used DOMContentLoaded to defer execution.

I actually hadn't considered using top level await. This is largely personal preference, but one of the things I like about leaning on DOMContentLoaded is that it clearly indicates when that initialization logic is executed. Without it, that timing is implicit based on attributes set outside the script's control (DOM order, script attributes, etc.).

According to Chrome Status, [top level await](https://www.chromestatus.com/feature/5767881411264512) isn't available until M89. For the moment I've wrapped it in an async IIFE.

Reply:
`DOMContentLoaded` isn't as useful as you think: https://whistlr.info/2020/understanding-load/#use-cases

Thanks for removing it though. Just running the code is clearer.